source/Octopus.Tentacle/Commands/ShowConfigurationCommand.cs

```diff
@@ -86,26 +87,14 @@ async Task StartAsync()
 
         internal async Task CollectConfigurationSettings(DictionaryKeyValueStore outputStore)
```

Comment:
This method was a pain in the ass - The `HomeConfiguration` and `TentacleConfiguration` classes would take a Configuration store (`IKeyValueStore`) but also an instance selector which is what you're meant to use to get the `IKeyValueStore`. But it was used to collect the properties to output them in JSON format. It's just very unclear where values are sourced from  and where values are set. I think I've made it clearer by using the `WriteTo` method so we know that the `IKeyValueStore` from the selector is used to source the data and it's written to the outputStore.